sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -1278,6 +1300,9 @@ struct _pi_event : _pi_object {
   // This list must be destroyed once the event has signalled.
   _pi_ze_event_list_t WaitList;
 
+  // Command list associated with the pi_event.
+  std::optional<pi_command_list_ptr_t> CommandList;
```

Comment:
if you have this way of knowing the command-list where the event belongs, why do you need `contains` method above?

Reply:
contains() is used to check whether _pi_ze_event_list_t contains some event. So, it is not related to this data member.

Reply:
ah, I see this is the list of events from commands, and there it is for wait-lists. ok